libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/NSURLResponse+SFAdditions.m

```diff
@@ -37,7 +37,8 @@ - (NSDictionary *)asDictionary {
     responseDictionary[@"textEncodingName"] = self.textEncodingName;
     responseDictionary[@"URL"] = self.URL.absoluteString;
     if ([self isKindOfClass:[NSHTTPURLResponse class]]) {
-        responseDictionary[@"allHeaderFields"] = ((NSHTTPURLResponse *)self).allHeaderFields;
+        responseDictionary[@"headers"] = ((NSHTTPURLResponse *)self).allHeaderFields;
```

Comment:
I noticed that the header name capitalization is different, Android is "content-type" vs iOS "Content-Type", should we make the iOS header keys lowercase?

Reply:
Nah - should be fine.

Reply:
It's also Content-Type on [Android](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestResponse.java#L52), as it [should be](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type), isn't it?

Reply:
Interesting -- when going through .getAllHeaders they're lowercase
![image](https://user-images.githubusercontent.com/840431/87609187-b3bbd380-c6b6-11ea-89c6-509b7497b34a.png)

Reply:
They are supposed to be case insensitive according to https://stackoverflow.com/a/5259004/11461823
Looking at the implementation on okHttp, it's indeed ignoring the case when getting headers by name: https://github.com/square/okhttp/blob/master/okhttp/src/main/kotlin/okhttp3/Headers.kt#L102